From dca7e418a40bc7e2a902ee8bd1f34023fdde456d Mon Sep 17 00:00:00 2001
From: TekkamanV <tekkamanv@starfivetech.com>
Date: Mon, 5 Jul 2021 23:04:29 +0800
Subject: [PATCH 47/62] media: starfive: Remove MODULE_SUPPORTED_DEVICE macro
 call

This macro was never implemented and completely removed upstream.

Signed-off-by: TekkamanV <tekkamanv@starfivetech.com>
---
 drivers/media/platform/starfive/stf_event.c | 1 -
 drivers/media/platform/starfive/stf_vin.c   | 1 -
 2 files changed, 2 deletions(-)

diff --git a/drivers/media/platform/starfive/stf_event.c b/drivers/media/platform/starfive/stf_event.c
index 088354fd700f..b4f5dfc3abbf 100644
--- a/drivers/media/platform/starfive/stf_event.c
+++ b/drivers/media/platform/starfive/stf_event.c
@@ -36,4 +36,3 @@ EXPORT_SYMBOL_GPL(vin_notifier_call);
 MODULE_AUTHOR("StarFive Technology Co., Ltd.");
 MODULE_DESCRIPTION("Starfive VIC video in notifier");
 MODULE_LICENSE("GPL");
-MODULE_SUPPORTED_DEVICE("video");
diff --git a/drivers/media/platform/starfive/stf_vin.c b/drivers/media/platform/starfive/stf_vin.c
index 711150399f6c..21fc51573f2c 100644
--- a/drivers/media/platform/starfive/stf_vin.c
+++ b/drivers/media/platform/starfive/stf_vin.c
@@ -932,4 +932,3 @@ module_exit(stf_vin_cleanup);
 MODULE_AUTHOR("StarFive Technology Co., Ltd.");
 MODULE_DESCRIPTION("Starfive VIC video in driver");
 MODULE_LICENSE("GPL");
-MODULE_SUPPORTED_DEVICE("video");
-- 
2.30.2

